tests/kernel_bundle/kernel_bundle.h

```diff
@@ -79,6 +79,36 @@ sycl_cts::util::kernel_restrictions get_restrictions() {
  */
 enum class submit_kernel { yes, no };
 
+/** @brief Call sycl::queue::submit only if dev_is_compat flag is true
```

Comment:
This is moved to avoid massive number of warnings of forward declaration available only in C++20.